src/commands/pjr.ts

```diff
@@ -3,12 +3,12 @@ import { registerProviders } from "../initialization/registerProviders";
 import { agile, project, links, playbook } from "./commands";
 
 export const pjr = new Command()
-  .description("Projector CLI")
+  .description("Root command for the Projector CLI")
   .name("pjr")
   .initialize(() => {
     registerProviders();
   })
-  .asciiArt("PJR")
+  .asciiArt("projector")
```

Comment:
This is cool! I didn't know this plugin existed, but it makes sense why I see that so often (and it always looks really cool)